jpo-ode-svcs/src/main/resources/application.yaml

```diff
@@ -1,4 +1,18 @@
 spring:
+  application:
+    name: "jpo-ode-svcs"
+  kafka:
+    consumer:
+      enable-auto-commit: true
+      auto-commit-interval: 1000
+      bootstrap-servers: ${DOCKER_HOST_IP}:9092
```

Comment:
note: pulled from https://github.com/CDOT-CV/jpo-ode/blob/96a889344c4eb6c8dce364513e536fd9931aa32f/jpo-ode-core/src/main/java/us/dot/its/jpo/ode/wrapper/MessageConsumer.java#L34-L36

Reply:
That could be an excellent addition, for sure! We'd want to ensure that in-order processing isn't required before adding concurrency since adding more consumers to a group can result in some messages being processed out of order in terms of time received.